server/services/prismic-parsers.js

```diff
@@ -317,7 +317,11 @@ export function asText(maybeContent: any) {
 }
 
 export function asHtml(maybeContent: any) {
-  return maybeContent && RichText.asHtml(maybeContent).trim();
+  // Prismic can send us empty html elements which can lead to unwanted UI in templates.
+  // Check that `asText` wouldn't return an empty string.
+  const isNonEmpty = maybeContent && asText(maybeContent).trim() !== '';
```

Comment:
To avoid exploding my tiny mind with double confirmations, would it be more clear to have:
```JS
const isEmpty = maybeContent && asText(maybeContent).trim() === '';
return isEmpty ? null : RichText.asHtml(maybeContent).trim();
```

Reply:
yeah I can get with that – will update